class OnPurposeJSError extends Error {}

throw new OnPurposeJSError("testing!");
